rpc/gnmi/gnmi-specification.md

```diff
@@ -917,7 +917,9 @@ The `GetResponse` message consists of:
     and hence MUST NOT collapse data from multiple paths into a single
     `Notification` within the response. The `timestamp` field of the
     `Notification` message MUST be set to the time at which the target's
-    snapshot of the relevant path was taken.
+    snapshot of the relevant path was taken. If the path is syntactically valid
+    but does not (yet) exist, then the `update` field of the `Notification` MUST
```

Comment:
I think here the target should respond with an error using the `NotFound` status code. This seems (to me) to be the most explicit thing to do when a client is explicitly asking for a very particular path.

Reply:
Wouldn't this deviate from Subscribe where no error is produced?

> There is no requirement that the path specified in the message must exist within the current data tree on the server. While the path within the subscription SHOULD be a valid path within the set of schema modules that the target supports, subscribing to any syntactically valid path within such modules MUST be allowed. In the case that a particular path does not (yet) exist, the target MUST NOT close the RPC, and instead should continue to monitor for the existence of the path, and transmit telemetry updates should it exist in the future.